resources/test_data/sqlite_testrunner_queries.sql

```diff
@@ -10,6 +10,7 @@ SELECT 22 / 5 AS col;
 
```

Comment:
(Random line)
@mrks, maybe @Bensk1 

I just removed the `TableStatistics::approx_invalid_row_count` property, so now ValidateNodes will always be estimated with a selectivity of "1".

Reasons are: The property was only manipulated in `Delete`, there was no equivalent in `Insert` so this wouldn't have worked for OLTP anyway. I want us to think of a proper approach for updating statistics once TPC-C support is there, not carry this pseudo-excuse property around. 
Also handing `approx_invalid_row_count` around in the `CardinalityEstimator` is cumbersome. Needs to be computed for the result of Predicates and Joins.
Adding it, or something similar, back in will be easy enough once we have a basis of discussion (cf TPC-C), but I am kind of fed up with keeping in speculative concepts that try to account for a future scenario that we have never thought about properly.

What do you think? Can this property go for now?

Reply:
> there was no equivalent in Insert

The only reason insert would have to update this is if a transaction with an insert failed. With optimistic concurrency control, we can be optimistic that this does not happen.

I expect a validate that happens always at the end to be bad for performance with the TPC-C. My goal is to clear up the TPC-C code enough over the weekend so that we can test this PR with the TPC-C, too.

Reply:
Any news here? Before TPC-C I would have thought the fixed 1.0 selectivity is sufficient (at least now that we no longer estimate some string predicates with 1.0 as well) as well. 

Do you already have numbers @mrks ?